protocol/communities/community_encryption_key_action.go

```diff
@@ -63,25 +63,25 @@ func evaluateCommunityLevelEncryptionKeyAction(origin, modified *Community, chan
 func evaluateChannelLevelEncryptionKeyActions(origin, modified *Community, changes *CommunityChanges) *map[string]EncryptionKeyAction {
 	result := make(map[string]EncryptionKeyAction)
 
-	for chatID := range modified.config.CommunityDescription.Chats {
-		originChannelViewOnlyPermissions := origin.ChannelTokenPermissionsByType(chatID, protobuf.CommunityTokenPermission_CAN_VIEW_CHANNEL)
-		originChannelViewAndPostPermissions := origin.ChannelTokenPermissionsByType(chatID, protobuf.CommunityTokenPermission_CAN_VIEW_AND_POST_CHANNEL)
+	for channelID := range modified.config.CommunityDescription.Chats {
+		originChannelViewOnlyPermissions := origin.ChannelTokenPermissionsByType(modified.IDString()+channelID, protobuf.CommunityTokenPermission_CAN_VIEW_CHANNEL)
```

Comment:
nit: you can put `modified.IDString()+channelID` in a variable and use that instead of concatenating each time.

Reply:
Right, done :+1: